doc/contributing/maintaining-openssl.md

```diff
@@ -7,15 +7,12 @@ currently need to generate four PRs as follows:
 
 * a PR for master which is generated following the instructions
   below for OpenSSL 3.x.x.
+* a PR for 18.x following the instructions in the v18.x-staging version
+  of this guide.
```

Comment:
I don't think a PR for v18.x is necessary, it's not very likely there will be a commit with `master`, most likely the same commit can be cherry-picked like any other non-semver-major commit, right?

Reply:
I'm adding it just to not forget when a new major version comes. Maybe not necessary then?

Reply:
Thanks for the clarification. Closing.